karma.entry.ts

```diff
@@ -3,7 +3,11 @@ import * as s from 'underscore.string';
 
 beforeEach(() => {
   if (!$('#App').length) {
-    $('body').append('<div id="App" class="coveo-styleguide"></div>');
+    $('body').append('<div id="App" class="coveo-styleguide">' +
+      '<table id="AppTable">' +
```

Comment:
I don't think this should be in the karma.entry file. It should be append where required instead.